File: zhiyoufy-server-java/zhiyoufy-server/src/main/java/com/example/zhiyoufy/server/domain/dto/ems/EmsEnvironmentUserRelationFull.java
package com.example.zhiyoufy.server.domain.dto.ems;

import com.example.zhiyoufy.mbg.model.EmsEnvironmentUserRelation;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class EmsEnvironmentUserRelationFull extends EmsEnvironmentUserRelation {
	private String username;
}

File: zhiyoufy-web/src/model/dto/ems/EmsEnvironmentUserRelationFull.ts
import type { EmsEnvironmentUserRelation } from './EmsEnvironmentUserRelation';


export interface EmsEnvironmentUserRelationFull extends EmsEnvironmentUserRelation {
  username: string;
}

export type EmsEnvironmentUserRelationParam = Omit<EmsEnvironmentUserRelationFull, 'id'>;
